### PR TITLE
Added mute and unmute command

### PR DESCRIPTION
**Pull request by 0xRedHeaD**

### CODE REVIEW

**Comment by itsmebirdie** [comment]
Acknowledged.
Assigning @mrnightdev

File: commands/mod/mute.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+module.exports = class MuteCommand extends Command {
+    constructor(client) {
+        super(client, {
+            name: 'mute',
+            group: 'mod',
+            memberName: 'mute',
+            description: 'Mute the desired member',
+            guildOnly: true,
+            clientPermissions: [
+                'MANAGE_MESSAGES'
```
Comment (mrnightdev) [inline commands/mod/mute.js:13]: Did you mean `MANAGE_ROLES` here?

File: commands/mod/unmute.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            name: 'unmute',
+            group: 'mod',
+            memberName: 'unmute',
+            description: 'Unmute the muted member',
+            guildOnly: true,
+            clientPermissions: [
+                'MANAGE_ROLES'
+            ],
+            userPermissions: [
+                'MANAGE_MESSAGES'
```
Comment (mrnightdev) [inline commands/mod/unmute.js:16]: I think you mean `MANAGE_ROLES`?

File: commands/mod/unmute.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            ]
+        });
+
+    }
+    run(message) {
+        const target = message.mentions.users.first();
```
Comment (mrnightdev) [inline commands/mod/unmute.js:22]: This will return a `TypeError` if someone runs the command without and arguments

**Review by mrnightdev — CHANGES_REQUESTED** [review]
Please look into the comments I have posted and make the appropriate changes.

**Comment by itsmebirdie** [comment]
Acknowledged and assigning myself due to @mrnightdev 's unavailability at the time.

**Pull request merged**